examples/crm/src/tasks/TaskEdit.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+    SimpleForm,
+    Edit,
+    TextInput,
+    required,
+    SelectInput,
+    DateInput,
+} from 'react-admin';
+import { Dialog, Stack } from '@mui/material';
+import { taskTypes } from './task.const';
+
+export const TaskEdit = ({
+    id,
+    setTaskSelectedId,
+}: {
+    id: string | undefined;
+    setTaskSelectedId: (id: string | undefined) => void;
+}) => {
+    const handleClose = () => {
+        setTaskSelectedId(undefined);
+    };
+
+    return (
+        <Dialog open={!!id} onClose={handleClose} fullWidth maxWidth="lg">
+            {!!id ? (
+                <Edit
```

Comment:
You should use `EditBase` instead. Having two page components in the same page causes conflicts in TitlePortal.